### ISSUE TITLE
How do I change AI faction to BLufor and Opfor and Player Faction to IND?

### ISSUE THREAD

**Issue by KRDucky** [brokenhands/ALiVEwACEteland.Altis#13]
How do I reverse your Faction change? I removed all the BLU/OPF players in the editor and placed IND faction players as well as set them to playable but, in the server, there is only 1 INDy slot. Not the 48 or so.  I dont see anything in the commits indicating the change.

**Comment by brokenhands** [comment]
Yeah github is going to suppress all the unit changes that happen in mission.sqm, it's just a shit file format IMO. 

As far as changing the playable units from BLU to IND, I really couldn't offer a ton of help. I'd recommend changing the unit type on each of them in the editor instead of removing them. Make sure to adjust the ALiVE modules as well, there are a few scripts that have AI that are spawned as IND that should get changed to BLU as well.

**Comment by KRDucky** [comment]
can you point to the scripts you changed? And how do you mean change the Unit type in the editor instead of removing them? I can not change the side without removing them. Which ALiVE modules should I be adjusting? I already changed the Commanders and Mil/Custom objectives accordingly.
